pkgs/tools/package-management/pdm/default.nix

```diff
@@ -7,22 +7,13 @@ let
     # 3. Ansible being unable to upgrade to a later version of resolvelib
     # see here for more details: https://github.com/NixOS/nixpkgs/pull/155380/files#r786255738
     packageOverrides = self: super: {
-      installer = super.installer.overridePythonAttrs (attrs: rec {
-        version = "0.6.0";
-        src = fetchFromGitHub {
-          owner = "pradyunsg";
-          repo = "installer";
-          rev = version;
-          hash = "sha256-IXznSrc/4LopgZDGFSC6cAOCbts+siKpdl5SvN1FFvA=";
-        };
-      });
```

Comment:
If you need 0.6.0, as opposed to 0.7.0, please keep this in. There will be another python updates run shortly, and we'll arrive at 0.7.0 for the release.

Reply:
pdm uses `installer >= 0.7` now

Reply:
And we already have 0.7.0. Sorry for the noise.